index.js

```diff
@@ -40,5 +41,6 @@ app.post('/api/setMapping', function(req, res) {
     timestampCreated: req.body.timestampCreated,
     timestampSent: req.body.timestampSent,
   };
```

Comment:
Not necessarily for this PR, but would be good to sanitize and validate the strings to make sure they're just text and not HTML, escaped characters, or some other weirdness designed to hack the system.

Reply:
Great point, made into issue